src/olympia/reviewers/templates/reviewers/emails/base.ltxt

```diff
@@ -7,6 +7,6 @@ If you want to respond please reply to this email or visit {{ dev_versions_url }
 You can also join us in #addon-reviewers on irc.mozilla.org.
 
 To learn more about the review process, please visit https://developer.mozilla.org/en-US/Add-ons/AMO/Policy/Reviews
---
+-- 
```

Comment:
added a trailing space?

Reply:
Yes, because [proper signature etiquette](https://en.wikipedia.org/wiki/Signature_block) dictates that the separator should be two dashes and one space :)

Reply:
trailing spaces 🙄